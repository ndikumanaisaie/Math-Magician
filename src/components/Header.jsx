import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
  <div className="nav-bar">
    <h2>Math Magician</h2>
    <div className="nav">
      <Link to="/">Home</Link>
      <span className="vertical-bar" />
      <Link to="calculator">Calculator</Link>
      <span className="vertical-bar" />
      <Link to="quotes">Quotes</Link>
    </div>
  </div>
);

export default Header;
