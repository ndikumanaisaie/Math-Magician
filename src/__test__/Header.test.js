import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import Header from '../components/Header';

it('renders correctly Header component', () => {
  const header = renderer.create(
    <MemoryRouter>
      <Header />
    </MemoryRouter>,

  ).toJSON();
  expect(header).toMatchSnapshot();
});
