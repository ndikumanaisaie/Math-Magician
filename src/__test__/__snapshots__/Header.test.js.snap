// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Header component 1`] = `
<div
  className="nav-bar"
>
  <h2>
    Math Magician
  </h2>
  <div
    className="nav"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <span
      className="vertical-bar"
    />
    <a
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <span
      className="vertical-bar"
    />
    <a
      href="/quotes"
      onClick={[Function]}
    >
      Quotes
    </a>
  </div>
</div>
`;
